### PR TITLE
Setup project template

### PR DESCRIPTION
**Pull request by talexandru1987**

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+:root {
```
Comment (surajverma2587) [inline assets/css/styles.css:1]: Not sure if this is just a placeholder for now or the finalised colour scheme for the project. Please discuss with your team to confirm the colours.

Comment (surajverma2587) [inline assets/css/styles.css:1]: Not sure if this is just a placeholder for now or the finalised colour scheme for the project. Please discuss with your team to confirm the colours.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <!-- link to bootstrap css-->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <!-- link to the css file-->
+    <link rel="stylesheet" href="./assets/css/styles.css" />
+    <title>Document</title>
```
Comment (surajverma2587) [inline index.html:25]: Please change the title to something more descriptive.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <!-- link to google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400;600;700&family=Josefin+Sans:wght@400;500;700&family=Lato:wght@100;400;700&family=Poppins:wght@300;400;700&family=Roboto+Condensed:wght@300;400;700&family=Roboto:wght@400;500;700&family=Smokum&family=Source+Sans+Pro:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- link to bootstrap css-->
+    <link
```
Comment (surajverma2587) [inline index.html:17]: Please import the latest version of bootstrap. See docs [here](https://getbootstrap.com/docs/5.1/getting-started/introduction/)

Comment (surajverma2587) [inline index.html:17]: Please import the latest version of bootstrap. See docs [here](https://getbootstrap.com/docs/5.1/getting-started/introduction/)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <link rel="stylesheet" href="./assets/css/styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main>
+      <p>hello</p>
+    </main>
+
+    <!-- link to jquery library-->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
```
Comment (surajverma2587) [inline index.html:33]: Please import the latest version of jQuery from [here](https://cdnjs.com/libraries/jquery)

**Comment by surajverma2587** [comment]
Linked a ticket on your board to the this PR, please see tasks and ensure that all is complete before merging this in

**Review by surajverma2587 — CHANGES_REQUESTED** [review]
Please address changes before merging so that we have a finalised template which we could use for all pages and ensure consistency moving forward.

File: index.html (hunk 4 above)
Comment (talexandru1987) [reply index.html:33]: updated

File: index.html (hunk 3 above)
Comment (talexandru1987) [reply index.html:17]: Will handle the project without bootstrap

File: assets/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  --aquamarine: #80ffdbff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Dancing Script", cursive;
```
[the remark this answers is not in the document]
Comment (talexandru1987) [reply assets/css/styles.css:21]: Fonts confirmed with the team

**Review by roxywasiak — APPROVED** [review]
yes we have looked at the document together and have agreed on the whole template.

**Review by katarzynajk — APPROVED** [review]
we discussed the changes and we are satisfied with them

**Pull request merged**